services/workflows-service/scripts/workflows/kyb-kyc-workflow-definition.ts

```diff
@@ -0,0 +1,280 @@
+      {
+        name: 'invitation_email',
+        pluginKind: 'email',
+        url: `{secret.EMAIL_API_URL}`,
+        method: 'POST',
+        stateNames: ['pending_invitation_email'],
+        successAction: 'INVITATION_EMAIL_SENT',
+        errorAction: 'INVITATION_EMAIL_FAILED',
+        headers: {
+          Authorization: 'Bearer {secret.EMAIL_API_TOKEN}',
+          'Content-Type': 'application/json',
+        },
+        request: {
+          transform: [
+            {
+              transformer: 'jmespath',
+              mapping: `{
+              templateId: 'd-00a0d5d14cb14fbb9034b53c6ef7e5fa',
+              adapter: '${env.MAIL_ADAPTER}'
+              from: 'no-reply@ballerine.com',
+              receivers: [mainRepresentative.email],
+              name: mainRepresentative.fullName,
+              provider: customerName,
+              url: join('',['${env.KYB_EXAMPLE_CORS_ORIGIN}?token=',token])
+              }`, // jmespath
+            },
+          ],
+        },
+        response: {
+          transform: [],
+        },
+      },
+      {
+        name: 'open_corporates',
+        pluginKind: 'kyb',
+        url: `${env.UNIFIED_API_URL}/companies`,
+        method: 'GET',
+        stateNames: ['run_kyb_enrichment'],
+        successAction: 'KYB_DONE',
+        errorAction: 'KYB_DONE',
+        headers: { Authorization: 'Bearer {secret.UNIFIED_API_TOKEN}' },
+        request: {
+          transform: [
+            {
+              transformer: 'jmespath',
+              mapping: `{
+              countryOfIncorporation: entity.data.countryOfIncorporation,
+              companyNumber: entity.data.registrationNumber,
+              state: entity.data.dynamicInfo.companyInformation.state
+              vendor: 'open-corporates'
+              }`, // jmespath
+            },
+          ],
+        },
+        response: {
+          transform: [
+            {
+              transformer: 'jmespath',
+              mapping: '@', // jmespath
+            },
+          ],
+        },
+      },
+      {
+        name: 'resubmission_email',
+        pluginKind: 'email',
+        url: `{secret.EMAIL_API_URL}`,
+        method: 'POST',
+        stateNames: ['pending_resubmission'],
+        headers: {
+          Authorization: 'Bearer {secret.EMAIL_API_TOKEN}',
+          'Content-Type': 'application/json',
+        },
+        request: {
+          transform: [
+            {
+              transformer: 'jmespath',
+              mapping: `{
+              kybCompanyName: entity.data.companyName,
+              customerCompanyName: entity.data.additionalInfo.ubos[0].entity.data.additionalInfo.customerCompany,
+              firstName: entity.data.additionalInfo.mainRepresentative.firstName,
+              resubmissionLink: join('',['https://',entity.data.additionalInfo.ubos[0].entity.data.additionalInfo.normalizedCustomerCompany,'.demo.ballerine.app','/workflowRuntimeId=',workflowRuntimeId,'?resubmitEvent=RESUBMITTED']),
```

Comment:
is it hard coded only for demo env? `'.demo.ballerine.app'`

Reply:
It shouldn't be hard-coded